tests/nested/manual/remodel-simple/task.yaml

```diff
@@ -0,0 +1,106 @@
+summary: verify a simple UC20 remodel
+
```

Comment:
Could you please also add a `details:` paragraph explaining what remodelling scenarios are we testing depending on the host system?